docs/content/latest/explore/cluster-management/point-in-time-recovery-ysql.md

```diff
@@ -31,11 +31,11 @@ showAsideToc: true
   </li>
 </ul>
 
-The point-in-time recovery feature allows you to restore the state of your cluster's data from a specific point in time. This can be relative, such as "three hours ago", or an absolute timestamp.
+The point-in-time recovery feature allows you to restore the state of your cluster's data (and some types of metadata) from a specific point in time. This can be relative, such as "three hours ago", or an absolute timestamp.
```

Comment:
```suggestion
The point-in-time recovery feature allows you to restore the state of your cluster's data (and [certain types](../../../manage/backup-restore/point-in-time-recovery/#limitations) of metadata) from a specific point in time. This can be relative, such as "three hours ago", or an absolute timestamp.
```